sources/packages/web/src/components/common/OfferingSummary.vue

```diff
@@ -224,12 +276,21 @@ export default defineComponent({
     const getEducationProgramAndOffering = async () => {
       try {
         loading.value = true;
+        const searchCriteria: Record<string, string> = {};
+        if (searchBox.value) searchCriteria.searchCriteria = searchBox.value;
```

Comment:
Since it's just used locally, can we rename `searchCriteria` to avoid having `searchCriteria.searchCriteria`?